kie-dmn-core/src/main/java/org/kie/dmn/core/ast/DecisionNode.java

```diff
@@ -147,7 +148,7 @@ public EvaluatorResult evaluate(InternalDMNRuntimeEventManager eventManager, DMN
                     params[i] = feel.evaluate( paramNames.get( i ), result.getContext().getAll() );
                     ctx.setValue( paramNames.get( i ), params[i] );
                 }
-                Object dtr = dt.apply( ctx, params );
+                Object dtr = dt.apply( ctx, params ).cata(e -> { events.add(e); return null; }, Function.identity());
```

Comment:
@etirelli kindly notice a small DMN-level fix is required, as FEEL.functions are not any longer returning a generic `Object` for result/null, but all aligned now using `Either<FEELEvent, Object>`